slices/libblkid1.yaml

```diff
@@ -0,0 +1,8 @@
+package: libblkid1
+
+slices:
+  libs:
+    essential:
+      - libc6_libs
+    contents:
+      /usr/lib/*-linux-gnu/libblkid.so.1*:
```

Comment:
Note that not all platform triplets end with `-gnu`:

| Arch | Triplet |
| --- | --- |
| amd64 | x86_64-linux-gnu |
| arm64 | aarch64-linux-gnu |
| armhf | arm-linux-gnueabihf |
| i386 | i386-linux-gnu |
| ppc64el | powerpc64le-linux-gnu |
| riscv64 | riscv64-linux-gnu |
| s390x | s390x-linux-gnu |

All shared library paths should contain wildcard for third triplet element as well, e.g.:
```
/usr/lib/*-linux-*/libblkid.so.1*
/lib/*-linux-*/libexpat.so.1*:

```

Reply:
Good catch. Done